### PR TITLE
added fetch for alerts and data processer

### PR DESCRIPTION
**Pull request by Fewwy**
I am adding fetch for alerts so I can process the data and prepare it for the graph
I added a data processer that converts timestamps into dates readable by human

### CODE REVIEW

File: src/components/console/utils/safe-fetch-hook.ts [hunk 1]
```diff
@@ -9,5 +9,5 @@ export const useSafeFetch = () => {
   }, []);
 
   return (url) =>
-    consoleFetchJSON(url, 'get', { signal: controller.current.signal as AbortSignal });
+    consoleFetchJSON(url, 'get', { signal: controller.current?.signal as AbortSignal });
```
Comment (Fewwy) [inline src/components/console/utils/safe-fetch-hook.ts:12]: @zhuje @kyoto For some reason this is undefined so I had to add an optional chain here. Do you have any idea why it is happening?

File: src/components/Incidents/IncidentsPage.jsx [hunk 2]
```diff
@@ -1,9 +1,107 @@
...
+      const promiseMap = _.map(timeRanges, (timeRange) =>
+        safeFetch(
+          getPrometheusURL(
+            {
+              endpoint: PrometheusEndpoint.QUERY_RANGE,
+              endTime: timeRange.endTime,
+              namespace,
+              query,
+              samples: Math.ceil(samples / timeRanges.length),
+              timeout: '60s',
```
Comment (jgbernalp) [inline src/components/Incidents/IncidentsPage.jsx:66]: should we leave this empty so the timeout defaults to the one configured in prometheus?

File: yarn.lock
Comment (jgbernalp) [inline yarn.lock:1]: Did you use npm instead of yarn? We will migrate out of yarn, but the current code still uses yarn

**Pull request merged**